Gemfile.lock

```diff
@@ -861,7 +854,6 @@ DEPENDENCIES
   equivalent-xml
   factory_bot_rails (~> 4.11.1)
   fakefs (~> 0.18.0)
-  fakeweb!
```

Comment:
Good 😄 but these changes should be applied also to the other Gemfile:
https://github.com/3scale/porta/blob/c567ceab6a86dd06f1dbdb34d006107e628c2010/gemfiles/prod/Gemfile.lock#L869